From 176546ed492883b63f6b8c274a97ce781af3631c Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Wed, 16 Aug 2017 23:35:07 +0200
Subject: Remove unused dependency

Summary:
libcurl doesn't seem to be used anymore since
427a9ede658e4bee0e96e09741adb879e396c0c1 and the removal of the gmic
plugin.

Test Plan: Still builds fine.

Reviewers: #krita, rempt

Subscribers: woltherav

Tags: #krita

Differential Revision: https://phabricator.kde.org/D7360
---
 CMakeLists.txt      | 10 ----------
 README_PACKAGERS.md |  3 ---
 2 files changed, 13 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5575b8c..b1edfc1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -604,16 +604,6 @@ set_package_properties(OpenMP PROPERTIES
     TYPE OPTIONAL
     PURPOSE "Optionally used by the G'Mic plugin")
 
-##
-## Test for Curl (for G'Mic)
-##
-find_package(CURL)
-set_package_properties(CURL PROPERTIES
-    DESCRIPTION "A tool to fetch remote data"
-    URL "http://curl.haxx.se/"
-    TYPE OPTIONAL
-    PURPOSE "Optionally used by the G'Mic plugin")
-
  ############################
 #############################
 ## Add Krita helper macros ##
diff --git a/README_PACKAGERS.md b/README_PACKAGERS.md
index ef66f9d..bd3d694 100644
--- a/README_PACKAGERS.md
+++ b/README_PACKAGERS.md
@@ -23,8 +23,6 @@ There are four forks of 3rd party libraries that are relevant and cannot be repl
 
 * plugins/impex/xcf/3rdparty contains the xcftools code. This has never been released as a library
 
-* plugins/extensions/gmic/3rdparty contains G'Mic. This has never been released as a shared library.
-
 * libs/image/3rdparty contains einspline. This code is directly linke d into the kritaimage library and has never been released as a separate library.
 
 == Build flags ==
@@ -47,7 +45,6 @@ Krita depends on:
   * ilmbase   
   * jpeg: Note that libjpeg-turbo is recommended.       
   * lcms2     
-  * libcurl   
   * libraw    
   * opencolorio
   * openexr   
-- 
cgit v0.11.2

